azkaban-webserver/src/main/resources/azkaban/webapp/servlet/velocity/executingflowpage.vm

```diff
@@ -112,6 +112,18 @@
         <li class="nav-button pull-right"><button type="button" id="cancelbtn" class="btn btn-danger btn-sm">Kill</button></li>
         <li class="nav-button pull-right"><button type="button" id="retrybtn" class="btn btn-success btn-sm">Retry Failed</button></li>
         <li class="nav-button pull-right"><button type="button" id="executebtn" class="btn btn-success btn-sm">Prepare Execution</button></li>
+        #*
+         * The 'pull-right' bootstrap class will cause the first 'li' element
+         * to take the right most spot. As 'analyzerButton' is mentioned in the
+         * last line, it will appear as the first button on the right.
+         *
+         * Tab focus is disabled on 'analyzerButton' by setting tabindex to -1 
+         * for the following reason. Whenever two buttons are displayed on the
```

Comment:
This will completely disable the ability to focus on the analyzer button, right? 

Would it be a better trade off to have the button show up next to the Stabs tab? This way the code reflects the same order as the order of the UI elements and tab order is more natural. 

Given that I don't expect keyboard based navigation to be widely used, I am more concerned about the code readability.